qiskit-superstaq/qiskit_superstaq/superstaq_provider.py

```diff
@@ -470,7 +470,10 @@ def cq_compile(
         )
 
     def supercheq(
-        self, files: List[List[int]], num_qubits: int, depth: int
+        self,
+        files: List[List[int]],
+        num_qubits: int,
+        depth: int,
```

Comment:
why this change? if unnecessary,
```suggestion
        self, files: List[List[int]], num_qubits: int, depth: int
```

Reply:
This was  the work of the formatting script, will revert.

Reply:
This still seems to be a problem (if I don't put the arguments on a new line, lint errors are thrown, looking into why this is the case now).